webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -1,29 +1,49 @@
 import React from 'react';
-import { EmptyState as PfEmptyState } from 'patternfly-react';
+import { Icon } from 'patternfly-react';
+import {
+  Title,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
 import { emptyStatePatternPropTypes } from './EmptyStatePropTypes';
+import { translate as __ } from '../../../common/I18n';
+import './EmptyState.scss';
 
 const EmptyStatePattern = props => {
   const {
-    documentation,
+    documentation: {
+      url,
+      label = __('For more information please see'),
+      buttonLabel = __('documentation'),
+    } = {},
     action,
     secondaryActions,
-    icon,
     iconType,
+    icon,
     header,
     description,
   } = props;
 
   return (
-    <PfEmptyState>
-      <PfEmptyState.Icon type={iconType} name={icon} />
-      <PfEmptyState.Title>{header}</PfEmptyState.Title>
-      <PfEmptyState.Info>{description}</PfEmptyState.Info>
-      {documentation && <PfEmptyState.Help>{documentation}</PfEmptyState.Help>}
-      {action && <PfEmptyState.Action>{action}</PfEmptyState.Action>}
-      {secondaryActions && (
-        <PfEmptyState.Action secondary>{secondaryActions}</PfEmptyState.Action>
-      )}
-    </PfEmptyState>
+    <EmptyState variant={EmptyStateVariant.xl}>
+      <span className="empty-state-icon">
+        <Icon name={icon} type={iconType} size="6x" />
+      </span>
+      <Title headingLevel="h5" size="4xl">
+        {header}
+      </Title>
+      <EmptyStateBody>
+        {description}
+        <br />
```

Comment:
imo we should use some css `margin` instead of `br`

Reply:
I tried using `<p>` instead of a margin, looks better now :+1:

Reply:
it's best not to have extra empty html elements,
we could wrap `description` with a span and add css margin to it.

Reply:
actually we could wrap it with a `div` and than what comes after it will be shifted down anyway
as `div` is a block element